scripts/in_container/run_generate_constraints.py

```diff
@@ -363,6 +367,7 @@ def generate_constraints_pypi_providers(config_params: ConfigParams) -> None:
                     f"a chicken egg provider"
                 )
                 packages_to_install.append(f"{provider_package} @ file://{file.as_posix()}")
+                continue
```

Comment:
Good point. I refactored it a bit - I think it was **slightly** wrong. The thing is that if we have chicken-egg provider, we should simply skip checkgin if it is available in PyPI - because it does not have to be in PyPI at all. It might be - if we release it earlier, but at the very initial phase when we just add a new provider, it will not be in PyPI, yet.

I added comment and moved continue outside of the inner for loop (where it should be in the first place and where it worked like that in `else` case.